docker/docker-compose.ci.yml

```diff
@@ -6,6 +6,14 @@ services:
     expose:
       - 8545
     command: ["-h=0.0.0.0", "-m=hello unlock save the web"]
+  db:
+    image: postgres
+    expose:
+      - 5432
+    environment:
+      POSTGRES_USER: "locksmith_test"
```

Comment:
I don't think this is a good idea, we can use Docker secrets and an environment variable for the user/password. I recognize this is a MVP user/password, but it's relatively trivial to set up Docker secret. We can talk tomorrow about it

Reply:
yeah i'd like to punt on this until after this is merged in as the deployment bits are currently shifting